parlai/chat_service/core/chat_service_manager.py

```diff
@@ -614,7 +629,28 @@ def _done_callback(fut):
             for agent in agents:
                 self.after_agent_removed(agent.id)
                 agent_state = self.get_agent_state(agent.id)
-                agent_state.set_active_agent(agent_state.get_overworld_agent())
+                next_task = agent.data.get("next_task")
+                shared_utils.print_and_log(
+                    logging.INFO, "Next task: {}".format(next_task)
+                )
+                if next_task is None:
```

Comment:
was this the default behavior before?

Reply:
Default behavior here in external ParlAI was to dump you into a completed overworld, which would just end a chat entirely. This reflects the default in our internal implementation.